schema_cache.rst

```diff
@@ -106,26 +106,52 @@ To refresh the cache without restarting the PostgREST server, send the server pr
      # or in docker-compose
      docker-compose kill -s SIGUSR1 <service>
 
-The above is the manual way to do it. To automate cache reloads, use a database trigger like this:
+Another option is to send a `database notification event <https://www.postgresql.org/docs/current/sql-notify.html>`_. To enable this feature, add this variable to the configuration file:
+
+.. code::
+
+  # postgrest.conf
+
+  db-channel-enabled = true
+
```

Comment:
Mentioning the config parameter in the paragraph gives us the opportunity to link:
```suggestion
Another option is to send a `database notification event <https://www.postgresql.org/docs/current/sql-notify.html>`_. To enable this feature, set db-channel-enabled(link here) to true.
```